src/main/osd/osd.c

```diff
@@ -189,16 +189,18 @@ const osd_stats_e osdStatsDisplayOrder[OSD_STAT_COUNT] = {
 };
 
 // Group elements in a number of groups to reduce task scheduling overhead
-#define OSD_GROUP_COUNT 20
+#define OSD_GROUP_COUNT                 OSD_ITEM_COUNT
 // Aim to render a group of elements within a target time
-#define OSD_ELEMENT_RENDER_TARGET 40
+#define OSD_ELEMENT_RENDER_TARGET       30
```

Comment:
Why lower render rate to 30?

Reply:
I was gathering some images of explanation when I spotted that the time isn't going where I thought it was in transfer to the MAX7456... I'll come back to you shortly on this.

Reply:
The reason for 30us is shown below on an F411 running an 8kHz PID loop.

The red `0` marker shows the cycle time from one gyro task call to the next as `schedulerExecuteTask()` is called. Following the gyro task can be seen the filter and PID tasks been called.

In the gaps is where we first run the checker routines and then schedule other tasks, and you can see groups of elements being rendered by calls to `osdDrawNextActiveElement()`. The below shows three groups targeting a max of 30 us as measured by the green `1` markers.

The purple `2` markers shows that there is only 5us spare. So 30us is a good value.

![image](https://user-images.githubusercontent.com/11480839/151246155-873a976a-23e7-4c50-9f2b-456dc22a6840.png)

Reply:
By comparison on an F722 the red 0 marker again shows the cycle time from one gyro task call to the next as schedulerExecuteTask() is called, but the proportion of time spent processing these is a little less despite bidirectional DSHOT being enabled in this case.

It can be seen that two groups of elements are rendered in a single gyro cycle as the first one completed early (render times can be rather variable depending on if they're interrupted by an ISR) leaving time for a second group.

The orange `3` markers show that 12 us remained which wouldn't be enough to render another group, although the final group had just been rendered in any case.
![image](https://user-images.githubusercontent.com/11480839/151248002-67d4fc79-b98b-4bef-9774-6dfa6aceedab.png)

Grouping 2 or 3 elements at a time (for the quicker to render elements at least) gives a reduction in scheduling overhead an so is more efficient.